EoT-Agents-Manifacturing-Marketplace/README.md

```diff
@@ -4,6 +4,7 @@
 
 - Setup of smart contracts and testnet as described [here](../service-directory/Readme.md).
 - [Fetchai](https://fetch.ai/) AEA version 1.1.1
+- Installation of [Ethereum ledger plugin](https://pypi.org/project/aea-ledger-ethereum/) in AEA using `pip install aea-ledger-ethereum` in the pipenv shell environment
```

Comment:
Actually this is installed through the [Pipfile](https://github.com/boschresearch/open-eot-agents/blob/8768450b3e86a1e012df6851276f0cb0d4f27302/EoT-Agents-Manifacturing-Marketplace/Pipfile#L8).

Reply:
Ah, you are right. Then we need to update that step in the Readme file, currently it is a mixture between all different approaches...

Reply:
I got my error, I had an additional Pipfile outside which I used...
Will update the Readme as well as the Pipfile to use aea 1.1.1